core/modules/layout/css/layout.admin.css

```diff
@@ -195,9 +214,11 @@
   float: left;
 }
 .layout-editor-block-content {
-  border: 1px solid #CCC;
+  background: #f7f7f7;
```

Comment:
there aren't any positioning or box-model styles? Only border, padding, height and font-size.